covid_graphs/nginx/nginx.conf

```diff
@@ -0,0 +1,66 @@
+worker_processes auto;
+pcre_jit on;
+
+events {
+    multi_accept on;
+    worker_connections 1024;
+}
+
+http {
+    charset utf-8;
+    sendfile on;
+    tcp_nopush on;
+    tcp_nodelay on;
+    server_tokens off;
+#    log_not_found off;
+    types_hash_max_size 2048;
+    client_max_body_size 256M;
+    aio threads;
+
+
+    include mime.types;
+    default_type application/octet-stream;
+
+    keepalive_timeout  65;
+
+    log_format forwarded '$remote_addr forwarded for $http_x_real_ip - $remote_user [$time_local] '
+                         '"$request" $status $body_bytes_sent '
+                         '"$http_referer" "$http_user_agent"';
+
+    access_log /var/log/nginx/access.log forwarded;
+    error_log /var/log/nginx/error.log warn;
+
+    gzip on;
+    gzip_vary on;
+    gzip_proxied any;
+    gzip_comp_level 6;
+    gzip_types text/plain text/css text/xml application/json application/javascript application/rss+xml application/atom+xml image/svg+xml;
+
+    server {
+        server_name _;
+        listen 80;
+        root /app/;
+        index index.html;
+
+        # favicon.ico
+        location = /favicon.ico {
+            log_not_found off;
+            access_log off;
+        }
+
+        # robots.txt
+        location = /robots.txt {
+            log_not_found off;
+            access_log off;
+        }
+
+        location / {
+            try_files $uri @proxy;
+        }
+
+        location @proxy {
+            include uwsgi_params;
+            uwsgi_pass dash_server:5000;
+        }
+    }
+}
```

Comment:
This config seems overly complicated for a server that sits in front of another Nginx -- see the drawing of how we would actually run it: ![Untitled drawing](https://user-images.githubusercontent.com/3822689/80825213-349f1f00-8be0-11ea-8646-064d45c2ac93.png)

I can copy some of these setting to the main Nginx if they are needed for the outside world.

Reply:
Config was simplified.